packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -99,7 +100,19 @@ export const mockHttpResponses = (options: NockBackOptions) => {
             })
             .get('/callback?usid=12345&code=ABCDE')
             .reply(200)
-
+        nock('http://localhost:3000')
+            .persist()
+            .post((uri) => {
+                return uri.includes('/oauth2/login')
+            })
```

Comment:
🤔 Can we really call `post` this way? The documentation is not super clear, but it looks to me that the first argument  can only be a string. The 2nd argument, though, can be a function: https://github.com/nock/nock#specifying-request-body

When we look at nock's source code, there are the arguments that `post` expects to see: https://github.com/nock/nock/blob/63668b3de22e140df025ef57072b715f9ee9bcc6/lib/scope.js#L110

Reply:
Not 100% sure, but would you expect this to be something like:
`.post('/oauth2/login//)
`?

Reply:
(i haven't read the docs yet) For `get`, you can pass in a string which is the path OR a matcher function. I'd imagine it's the same for `post`.